mealSharing-Nodejs/src/client/components/Meals.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+export default function Meals({meals}){
+  
+return(
+    <div>
+    <ul>{Meals.map((item)=>{
+        return (
+
+          <li>{item.title}</li>
```

Comment:
Here, you should also have a link which takes you to each meal's, url (meals/{id}).
Otherwise, you cannot visit that view, unless you manually type in a specific url at your browser's address bar, eg meals/1

Reply:
Hi, can you please review my code again?